sdk/core/core-http/review/core-http.api.md

```diff
@@ -248,28 +238,34 @@ export function executePromisesSequentially(promiseFactories: Array<any>, kickst
 // @public @deprecated
 export class ExpiringAccessTokenCache implements AccessTokenCache {
     constructor(tokenRefreshBufferMs?: number);
-    // (undocumented)
     getCachedToken(): AccessToken | undefined;
-    // (undocumented)
     setCachedToken(accessToken: AccessToken | undefined): void;
 }
 
-// @public (undocumented)
+// @public
 export function exponentialRetryPolicy(retryCount?: number, retryInterval?: number, maxRetryInterval?: number): RequestPolicyFactory;
 
-// @public (undocumented)
+// @public
+export abstract class FetchHttpClient implements HttpClient {
```

Comment:
Necessary because of a previous warning on the `DefaultHttpClient`:

```
// Warning: (ae-forgotten-export) The symbol "FetchHttpClient" needs to be exported by the entry point coreHttp.d.ts
```